src/widgets/component-new-edit/components/response-format/multiple/multiple-measure-codeslist.jsx

```diff
@@ -13,50 +13,42 @@ import GenericOption from 'forms/controls/generic-option';
 
 const { CHECKBOX, RADIO, DROPDOWN } = DATATYPE_VIS_HINT;
 const { CODES_LIST } = DIMENSION_FORMATS;
+const selectorPath = CODES_LIST;
```

Comment:
could destructure and rename on same line
could nest-destructure and rename directly in import line 9